### PR TITLE
Positive and Negative Handlers

### LINKED ISSUE #14993: assumptions fail on simple Q.Positive expression
sympy1.2: ask(Q.positive(-x-1), Q.positive(x)) returns None
This used to work (return False) on sympy 1.0

### PR DESCRIPTION
**Pull request by Arpan612**
#### References to other Issues or PRs

Fixes #14993 Closes #16213 

#### Brief description of what is fixed or changed
Improved the code of AskPositiveHandler and AskNegativeHandler.
Earlier

>>> from sympy import Q, ask
>>> from sympy.abc import x
>>> ask(Q.positive(-x-1), Q.positive(x))
>>> ask(Q.negative(-x+1), Q.negative(x))
>>>
But now

>>> ask(Q.positive(-x-1), Q.positive(x))
False
>>> ask(Q.negative(-x+1), Q.negative(x))
False
>>>

#### Other comments
Completion of work started in PR #16213 

#### Release Notes

* assumptions
   * Improve the positive and negetive handler

### CODE REVIEW

**Comment by Arpan612** [comment]
@jksuom @asmeurer Please tell me if there is anything else I need to act. The issue of Codecov tests is taken care of I think.

**Comment by asmeurer** [comment]
If this is based on another branch, you should start with the other branch and make new commits off of it. Otherwise, the commit history will show you as the author, rather than the true author.

File: sympy/assumptions/handlers/order.py [hunk 1]
```diff
@@ -31,10 +31,10 @@ def Expr(expr, assumptions):
     @staticmethod
     def _number(expr, assumptions):
         r, i = expr.as_real_imag()
-        # If the imaginary part can symbolically be shown to be zero then
-        # we just evaluate the real part; otherwise we evaluate the imaginary
-        # part to see if it actually evaluates to zero and if it does then
-        # we make the comparison between the real part and zero.
+        """If the imaginary part can symbolically be shown to be zero then
```
Comment (asmeurer) [inline sympy/assumptions/handlers/order.py:34]: Don't use strings for comments.

**Comment by Arpan612** [comment]
@asmeurer Yes sir. I will follow this method next issue onwards. I have requested @ShubhamKJha to make a commit so that he is also shown as co-author.

File: sympy/assumptions/handlers/order.py (hunk 1 above)
Comment (Arpan612) [reply sympy/assumptions/handlers/order.py:34]: Okay sir. @ShubhamKJha Can you make the change here? It will make you the co-author.

**Comment by ShubhamKJha** [comment]
Hi @Arpan612, please follow this method (https://github.com/sympy/sympy/pull/17144#issuecomment-511472242). It won't take time and would restore respective author's histories with proper attribution to each commit. Also, it would give mentors an insight into what changes you've brought over my PR.

**Comment by smichr** [comment]
this is continued in #19241

**Pull request closed without merging**